SCENE-SCRAPABLE.md

```diff
@@ -16,6 +16,7 @@ Supported Site|Scraper|Content
 50plusmilfs.com|TheScoreGroup.yml|
 60plusmilfs.com|TheScoreGroup.yml|Granny
 8thstreetlatinas.com|MindGeek.yml|
+AbbyWinters.com|AbbyWinters.yml|Lesbian
```

Comment:
Is Lesbian an accurate category here? It contains Lesbian content yes, but it does not seem to be an exclusively Lesbian site.